nifi/nifi-nar-bundles/nifi-flume-bundle/nifi-flume-processors/src/test/resources/testdata/records.txt

```diff
@@ -0,0 +1,4 @@
+record 1
```

Comment:
please either add a license header or add this file to the list of exemptions for apache-rat:check